solver/llbsolver/ops/exec_binfmt.go

```diff
@@ -106,7 +118,8 @@ func getEmulator(p *pb.Platform, idmap *idtools.IdentityMapping) (*emulator, err
 
 	fn, err := exec.LookPath("buildkit-qemu-" + a)
 	if err != nil {
-		return nil, errors.Errorf("no emulator available for %v", pp.OS)
```

Comment:
any reason to not error anymore?

Reply:
This log was previously in ops/exec.go . Previously we ignored all errors. Now only ignore this case.

Reply:
ok got it I missed the one in exec.